src/AccessControl/tainted.py

```diff
@@ -169,3 +188,19 @@ def createOneOptArgWrapper(func):
 
 for f in oneOptArgWrappedMethods:
     setattr(TaintedString, f, createOneOptArgWrapper(f))
+
+
+class TaintedBytes(TaintedString):
+    
+    def __init__(self, value):
+        if isinstance(value, int):
+            assert six.PY3
```

Comment:
Please no assert statements in non-test code:  they get "compiled away" when run with `-O`.

Instead, raise a `ValueError`.